x-pack/plugins/task_manager/server/config.ts

```diff
@@ -67,9 +73,17 @@ const requestTimeoutsConfig = schema.object({
 
 export const configSchema = schema.object(
   {
+    active_nodes_lookback: schema.number({
+      defaultValue: DEFAULT_ACTIVE_NODES_LOOK_BACK_S,
+      min: 10,
```

Comment:
not sure if we should set a max as well?

Reply:
Would 5 minutes make sense? If a user sets 5m, it would take 5m for a node that crashed to disappear from the list of active nodes. Perhaps its a sufficient high bound that we'd want to support?

Reply:
Updated in https://github.com/elastic/kibana/pull/190934/commits/730d9337a47ed52f57fe171e57bbb55fbfb0f920